[2024-08-27T01:24:50.956+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-27T01:24:51.270+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_operator_dag.create_table manual__2024-08-27T01:19:25.390676+00:00 [queued]>
[2024-08-27T01:24:51.360+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_operator_dag.create_table manual__2024-08-27T01:19:25.390676+00:00 [queued]>
[2024-08-27T01:24:51.361+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2024-08-27T01:24:51.410+0000] {taskinstance.py:2330} INFO - Executing <Task(SQLExecuteQueryOperator): create_table> on 2024-08-27 01:19:25.390676+00:00
[2024-08-27T01:24:51.447+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1293) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-27T01:24:51.454+0000] {standard_task_runner.py:63} INFO - Started process 1306 to run task
[2024-08-27T01:24:51.461+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'create_table', 'manual__2024-08-27T01:19:25.390676+00:00', '--job-id', '498', '--raw', '--subdir', 'DAGS_FOLDER/dag_creation.py', '--cfg-path', '/tmp/tmpcjby8sdq']
[2024-08-27T01:24:51.462+0000] {standard_task_runner.py:91} INFO - Job 498: Subtask create_table
[2024-08-27T01:24:51.586+0000] {task_command.py:426} INFO - Running <TaskInstance: postgres_operator_dag.create_table manual__2024-08-27T01:19:25.390676+00:00 [running]> on host 6a900cf20cab
[2024-08-27T01:24:51.752+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anderson_oca' AIRFLOW_CTX_DAG_ID='postgres_operator_dag' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-08-27T01:19:25.390676+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-27T01:19:25.390676+00:00'
[2024-08-27T01:24:51.753+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-27T01:24:51.784+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS fin_mundo (
            dt DATE,
            pais VARCHAR(30)
            );
          
[2024-08-27T01:24:51.798+0000] {base.py:84} INFO - Using connection ID '***_operator_dag' for task execution.
[2024-08-27T01:24:51.812+0000] {base.py:84} INFO - Using connection ID '***_operator_dag' for task execution.
[2024-08-27T01:24:51.823+0000] {sql.py:487} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS fin_mundo (
            dt DATE,
            pais VARCHAR(30)
            );
          , parameters: None
[2024-08-27T01:24:51.826+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-27T01:24:51.852+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***_operator_dag, task_id=create_table, run_id=manual__2024-08-27T01:19:25.390676+00:00, execution_date=20240827T011925, start_date=20240827T012451, end_date=20240827T012451
[2024-08-27T01:24:51.950+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-27T01:24:51.967+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
